packages/react-dom/src/test-utils/ReactTestUtils.js

```diff
@@ -39,7 +39,9 @@ const getFiberCurrentPropsFromNode = EventInternals[2];
 const enqueueStateRestore = EventInternals[3];
 const restoreStateIfNeeded = EventInternals[4];
 
-const act = React.unstable_act;
+// TODO: Add a warning if this API is accessed with advice to switch to
```

Comment:
If we want to remove react-dom/test-utils, do we really need a warning since that won't appear in any stable release anyway?

Reply:
I think the current plan is to remove everything except `act` from the `test-utils` for a single release, then remove in 20.